qa/tasks/cephfs/fuse_mount.py

```diff
@@ -418,8 +418,6 @@ def teardown(self):
         """
         super(FuseMount, self).teardown()
 
-        self.umount()
-
         if self.fuse_daemon and not self.fuse_daemon.finished:
             self.fuse_daemon.stdin.close()
             try:
```

Comment:
out of academic interest, shouldn't the base class teardown() happen **after** the derived class teardown() ?

Reply:
Yes. We live in a special circle of Python's Hell.